### ISSUE TITLE
Move ScriptContext and dependencies to own library/package

### ISSUE THREAD

**Issue by thebigb** [daveaglick/Scripty#57]
Related to #36.

It would be nice if there would not be a truckload of dependencies (Roslyn and friends) when you only need the `ScriptContext` type.

An issue would be the `ProjectRoot` type referenced in `ScriptContext`, because it has direct references to the `Microsoft.CodeAnalysis` namespace. `FormatterOptions` has some indirect references, but that could be solved with an interface.

The only way I see to solve this issue would be to remove `ProjectRoot` from `ScriptContext`, and to make it available separately in the global scope. This would be a breaking change, but perhaps a good one to make before a 1.0.0 release.

**Comment by daveaglick** [comment]
I'm curious what the use case would be for that? The dependencies provide the functionality to compile and evaluate the script, without them the script is just a random text file. Is the idea that you would use this lightweight library to provide Intellisense if you don't care about evaluation locally (presumably doing it on a build server or something)?

**Comment by thebigb** [comment]
Yes, this would indeed be for Intellisense, but now that you mention it, I
realize a stub could suffice.
